Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -376,4 +376,69 @@ public void Dispose()
             _context.Dispose();
         }
     }
+
+    //public class TheCreateReactionMethod : IDisposable
```

Comment:
It'd be cool to uncomment this test and see if we can get it working. I can help verify this (we don't run them on CI because they're hitting the actual API).